t/01-wasm/002-shm_queue_sanity.t

```diff
@@ -14,6 +14,7 @@ __DATA__
 
 === TEST 1: queue - with minimum size
 --- valgrind
+--- skip_eval: 5: ($::osname =~ m/darwin/ && $::archname =~ /arm64/)
```

Comment:
I don't know if you follow the changes in the main branch but since recently we don't specify the number of skipped tests in this block anymore.

Reply:
Yes, I follow the main branch I noticed those changes.

However, since I've seen many instances of `skip_eval` still specifying the number of tests to be skipped in the [commit](https://github.com/Kong/ngx_wasm_module/commit/a278bb7ce5da7f26f6faeb98352890313af05e9b#diff-80df4c463a1ccc8ebbfd5b1f3ec4cb380e77db0fe55dab23503ac3820cb81740L418) introducing the improvement, I assumed it didn't apply to `skip_eval`.

After reading you comment I tried removing the number of skipped tests from a few `skip_eval` instances but they didn't work.

Reply:
Hmmm ok then, let's just keep them like this for now and we'll look later what goes wrong without the number.